app/src/main/java/com/sopt/now/ui/main/MainActivity.kt

```diff
@@ -0,0 +1,73 @@
+package com.sopt.now.ui.main
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
+import com.sopt.now.R
+import com.sopt.now.databinding.ActivityHomeBinding
+import com.sopt.now.ui.home.HomeFragment
+import com.sopt.now.ui.myPage.MyPageFragment
+import com.sopt.now.ui.search.SearchFragment
+
+class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityHomeBinding
+    private lateinit var id: String
+    private lateinit var pw: String
+    private lateinit var nickname: String
+    private lateinit var mbti: String
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityHomeBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        id = intent.getStringExtra("id").toString()
+        pw = intent.getStringExtra("pw").toString()
+        nickname = intent.getStringExtra("nickname").toString()
+        mbti = intent.getStringExtra("mbti").toString()
+
```

Comment:
요거 한번 scope 함수로 묶어보실래욥?
```suggestion
        intent.run {
        id = getStringExtra("id").toString()
        pw = getStringExtra("pw").toString()
        nickname = getStringExtra("nickname").toString()
        mbti = getStringExtra("mbti").toString()
        }
       
```

Reply:
그리고 키값은 상수로!

Reply:
상수화 지적 감삼답!!!!